### DISCUSSION TITLE
Best way of implementing series

### DISCUSSION THREAD

**Discussion by Alextocode** [stashapp/stash#3980] [Q&A] [answered]
Hi there,
I installed an Ubuntu server and I'm running Stash as a Docker container there. Works like a charm in general. Thumbs up!
One question comes up and this is when it comes about series.

I will give the example:
There is a studio called X. They do have many "single videos" where there is no other video that is similar.
However they also have videos that are part of a series and are represented by multiple vidoes.
For instance there are series like A and B and A has multiple videos under X and B as well.

Currently I am implementing this by making my series A and B actually also studios and set X as the parent studio.
The disadvantage of this implementation is that when you are trying to work with the studio logos of course series usually do not have their own logos, only studios and maybe huge projects by a studio rarely have one.

I therefore wonder if there is a better way of implementing series of videos published by the same studio.

Any advice is highly appreciated.
Thanks so much!

Cheers!

Alex

**Comment by sleetx** [comment]
> I will give the example: There is a studio called X. They do have many "single videos" where there is no other video that is similar. However they also have videos that are part of a series and are represented by multiple vidoes. For instance there are series like A and B and A has multiple videos under X and B as well.

sounds that would be a couple of Movies? collections of related scenes created by one studio... You could create a movie by that studio, and then tie the scenes to it

**Comment by stg-annon** [comment, marked as answer]
The idea has been tossed around a few times about adding a *Collection* feature to Stash I cant seem to find any of the issues about it but I think it was in relation to Movies

A few approaches to handling this content:
- Tagging: you could create a unique tag for each series and use that to filter by 
- Studio: creating a sub studio under the parent studio for each series
- Movie: create a movie for each series and add each scene

I think I would personally go the studio route


Edit: 
Issue just got brought up in discord, Issue #2821 might be something to look at

**Reply by Alextocode** [reply]
Thanks so much for your input. I think I have to go the studio route.
The thing is it would be inconvenient if I do it this way now and in a soon to be released version the missing collection feature gets added....

Actually I am also a little familar with coding though I am not a developer but only an IT technician so I'd love to contribute here. On the other hand I am only familar with PHP, Java and PowerShell so I do not dare to touch code written in Go or TypeScript :(

**Reply by stg-annon** [reply]
any new features related to this would likely not be added anytime soon so you should be safe for now, contributions are welcome in any way plugins, scrapers, documentation whatever way you feel like.
